tests/testthat/test-step-call.R

```diff
@@ -161,3 +161,69 @@ test_that("unique is an alias for distinct", {
   dt <- lazy_dt(data.table(x = c(1, 1)))
   expect_equal(unique(dt), distinct(dt))
 })
+
+# drop_na ------------------------------------------------------------------
+
+test_that("empty call drops every row", {
+  df <- lazy_dt(tibble(x = c(1, 2, NA), y = c("a", NA, "b")), "DT")
+  exp <- tibble(x = c(1), y = c("a"))
+  step <- drop_na(df)
+  res <- as_tibble(step)
+  expect_equal(show_query(step), expr(na.omit(DT)))
+  expect_equal(res, exp)
+})
+
+test_that("converts data.table to dtplyr_step", {
+  df <- data.table(x = c(1, 2, NA), y = c("a", NA, "b"))
+  expect_equal(class(drop_na(df)), c("dtplyr_step_call", "dtplyr_step"))
+})
+
+test_that("specifying (a) variables considers only that variable(s)", {
+  df <- lazy_dt(tibble(x = c(1, 2, NA), y = c("a", NA, "b")), "DT")
+  exp <- tibble(x = c(1, 2), y = c("a", NA))
+  step <- drop_na(df, x)
+  res <- as_tibble(step)
+  expect_equal(show_query(step), expr(na.omit(DT, cols = "x")))
+  expect_equal(res, exp)
+  exp <- tibble(x = c(1), y = c("a"))
+  res <- as_tibble(drop_na(df, x:y))
+  expect_equal(res, exp)
+})
+
+test_that("groups are preserved", {
```

Comment:
Is it necessary to test this? It's `step_call()` that takes care of this, not your code.